migrations/educandu-2022-01-21-01-add-render-media-to-markdown-and-annotation.js

```diff
@@ -0,0 +1,59 @@
+// eslint-disable-next-line camelcase
+export default class Educandu_2022_01_21_01_add_render_media_to_markdown_and_annotation {
+  constructor(db) {
+    this.db = db;
+  }
+
+  async up() {
+    const updateFunc = section => {
+      section.content.renderMedia = false;
+    };
+    await this.updateSectionsInCollection('documentRevisions', ['markdown', 'annotation'], updateFunc);
+    await this.updateSectionsInCollection('documents', ['markdown', 'annotation'], updateFunc);
+  }
+
+  async down() {
+    const updateFunc = section => {
+      delete section.content.renderMedia;
+    };
+
+    await this.updateSectionsInCollection('documentRevisions', ['markdown', 'annotation'], updateFunc);
+    await this.updateSectionsInCollection('documents', ['markdown', 'annotation'], updateFunc);
+  }
+
+  async updateSectionsInCollection(collectionName, sectionTypes, updateFunc) {
+    const filter = { $or: sectionTypes.map(st => ({ 'sections.type': st })) };
```

Comment:
You could also filter more simply by `{ 'sections.type': { $in: sectionTypes } }`

Reply:
Of course, thanks, I didn't think of it. Changed